multicluster/test/e2e/README.md

```diff
@@ -0,0 +1,14 @@
+# Running the Antrea Multicluster end-to-end tests
+
+## Creating the test kubernetes cluster set
+
+The tests must be run on an actual Kubernetes cluster set, and there must be at least three clusters. We can create the clusters by [kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm) or the [instructions](https://github.com/antrea-io/antrea/blob/main/test/e2e/README.md) or other ways.
+
+The three clusters is called leader cluster, west cluster and east cluster. And the CIDR of pod in each clusters should not same. For example, the cidr of pod could be `192.168.0.1/22`,`192.168.4.1/22` and `192.168.8.1/22` in three clusters.
+
+## Running the tests
+
+Make sure that your cluster was provisioned and that the Antrea build artifacts were pushed to all the Nodes. If you install the multicluster manully, You can then run the tests from the top-level directory with `go test -v antrea.io/antrea/multicluster/test/e2e` or you can just run `bash ci/jenkins/test-mc.sh --testcase e2e`.
+
+When you use the `test-mc.sh`, make sure your kubeconfig files of the clusters in the specific path of the scripts, or you can change the parameter `MULTICLUSTER_KUBECONFIG_PATH` and other parameters to your own path in the scripts. 
```

Comment:
the `test-mc.sh` -> `test-mc.sh`

in the specific path of the scripts -> are in the path of the script?

the scripts -> the script